src/rstxml2db/rstxml2db.xsl

```diff
@@ -705,8 +705,60 @@
     </informalfigure>
   </xsl:template>
 
-  <xsl:template match="paragraph[strong][preceding-sibling::figure]"/>
+	<xsl:template match="paragraph[strong][preceding-sibling::figure]"/>
+
+	<xsl:template match="desc">
+    <xsl:variable name="id" select="normalize-space(desc_signature/@ids)"/>
+	  <variablelist>
+      <xsl:message>INFO: ID "<xsl:value-of select="$id"/>"</xsl:message>
+			<varlistentry><!-- Bug in libxslt with AVT? -->
+			<xsl:if test="$id != ''">
+			  <xsl:attribute name="xml:id">
+			     <xsl:value-of select="$id"/>
+			  </xsl:attribute>
+			</xsl:if>
+          <xsl:apply-templates/>
+			</varlistentry>
+		</variablelist>
+	</xsl:template>
+	
+	<xsl:template match="desc_signature">
+    <xsl:variable name="name">
+      <xsl:choose>
+        <xsl:when test="../@objtype = 'function'">
+          <xsl:text>function</xsl:text>
+        </xsl:when>
+        <xsl:when test="../@objtype = 'attribute'">
+          <xsl:text>property</xsl:text>
+        </xsl:when>
+        <xsl:when test="../@objtype='classmethod'">
+          <xsl:text>property role="classmethod"</xsl:text>
+        </xsl:when>
+        <xsl:when test="../@objtype='staticmethod'">
+          <xsl:text>property role="staticmethod"</xsl:text>
+        </xsl:when>
+        <xsl:otherwise>
+          <xsl:text>literal</xsl:text>
+        </xsl:otherwise>
+      </xsl:choose>
+    </xsl:variable>
+      <term>
+          <xsl:value-of select="@ids"/>
```

Comment:
The element name you created with the variable `name` is missing.  You need to correct the term with:

```xml
<term>
  <xsl:element name="{$name}">
     <xsl:value-of select="@ids"/>
  </xsl:element>
</term>
```